chapter_deep-learning-computation/deferred-init_vn.md

```diff
@@ -266,7 +270,10 @@ net.initialize(init=MyInit())
 3. What would you need to do if you have input of varying dimensionality? Hint - look at parameter tying.
 -->
 
-*dịch đoạn phía trên*
+1. Chuyện gì xảy ra nếu ta chỉ chỉ rõ một vài phần của chiều đầu vào? 
+Có thể vẫn khởi tạo ngay lập tức được không?
+2. Chuyện gì xảy ra nếu ta truyền vào giá trị chiều không phù hợp?
+3. Bạn cần làm gì nếu đầu vào có chiều biến thiên? Gợi ý - tìm hiểu về ràng buộc tham số (parameter tying).
```

Comment:
```suggestion
3. Bạn cần làm gì nếu đầu vào có chiều biến thiên? Gợi ý - tìm hiểu về ràng buộc tham số (*parameter tying*).
```

Reply:
Chỗ này là style guide của nhóm khi dịch một từ có kèm từ gốc thì mình in nghiêng từ gốc để trong ngoặc, em lưu ý nghen.

Reply:
@duythanhvn vâng ạ.